code/datums/movement/mob.dm

```diff
@@ -188,7 +188,7 @@
 			to_chat(mob, SPAN_WARNING("You're pinned down by \a [mob.pinned[1]]!"))
 		return MOVEMENT_STOP
 
-	for(var/obj/item/grab/G in mob.grabbed_by)
+	for(var/obj/item/grab/G as anything in mob.grabbed_by)
 		if(G.assailant != mob && (mob.restrained() || G.stop_move()))
```

Comment:
In testing things seem to work much better when this line is changed to `if(G.assailant != mover && (mob.restrained() || G.stop_move()))` (mob -> mover)

Reply:
If it is the case that a mob can grab itself (to hold wounds perhaps?) then it should likely instead be `if(G.assailant != mob && G.assailant != mover && (mob.restrained() || G.stop_move()))`